app/src/components/staking/Undelegations.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="table-container">
+  <div v-if="undelegations.length" class="table-container">
```

Comment:
shouldn't it be `v-if="undelegations.length > 0"` ?

Reply:
```suggestion
  <div v-if="undelegations.length > 0" class="table-container">
```